src/class-wpml-elementor-translatable-nodes.php

```diff
@@ -903,6 +903,24 @@ public static function get_nodes_to_translate() {
 					),
 				),
 			),
+			'galleries'           => array(
```

Comment:
The indentation is incorrect in your change @GeorgeBotsev (see https://github.com/OnTheGoSystems/wpml-page-builders-elementor/blob/9e13b1d205d0961474db519f2613d5ba8f6260ed/src/class-wpml-elementor-translatable-nodes.php#L906). You probably need to adjust your IDE settings (ping me if you want).

Reply:
This should be fixed now.